server/index.js

```diff
@@ -10,10 +10,10 @@ const bodyParser = require('body-parser');
 
 const app = express();
 
-app.use(express.static(`${__dirname}/../client/public`)); //Need to change this file path
+app.use(express.static(`${__dirname}/../client/dist`)); //Need to change this file path
 app.use(bodyParser.json());
 
-const port = process.env.PORT || 8080;
+const port = process.env.PORT || 8081;
```

Comment:
Do we need to change this back to 8080 or did we decide to stick w 8081?

Reply:
it doesnt matter